process_request/helpers.py

```diff
@@ -3,6 +3,70 @@
 from rapidfuzz import fuzz
 
 
+def get_collection_creator(archival_object):
```

Comment:
if you pass in the resource json directly it will save you some trouble here...see my comment below in the `get_data` method.